wasm/typescript/package.json

```diff
@@ -47,7 +47,7 @@
     "@itk-wasm/image-io": "^1.3.0",
     "@types/node": "^20.2.5",
     "ava": "^5.3.1",
-    "cypress": "^14.3.0",
+    "cypress": "^14.5.1",
```

Comment:
Is this commit (COMP: Bump cypress to 14.5.1) still useful? Isn't it superseded by the commit "ENH: Migrate browser test from cypress to playwright" in the very same PR?